website/fiddle/src/App.svelte

```diff
@@ -101,10 +105,30 @@
     }
   }
 
+  function initKeyboardEvents() {
```

Comment:
I think the function can be shortened. The shortened version works on OSX. Can't test it on win.
Why do we need the metaKeyEnabled variable?

```
function initKeyboardEvents() {
    window.addEventListener('keydown', (event) => {
      if ((event.ctrlKey || event.metaKey) && event.key === 's'){
        event.preventDefault();
        saveConfig();
      }
    });
  }
```